### PR TITLE
Include notes in the CVEs text field search

### PR DESCRIPTION
**Pull request by carkod**
## Done

Allow notes to be searchable with the text field. This is an OR type of search.

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: http://0.0.0.0:8030/security/cves.json?q=sql should show results from the notes section.
- Ideally test with prod data
- Tests should pass

## Issue / Card

Fixes https://warthogs.atlassian.net/browse/WD-11712

### CODE REVIEW

File: scripts/generate-sample-security-data.py [hunk 1]
```diff
@@ -41,7 +41,7 @@
                 published=datetime.now(),
                 description="",
                 ubuntu_description="",
-                notes={},
+                notes=[],
```
Comment (carkod) [inline scripts/generate-sample-security-data.py:44]: This causes errors coming from Postgres
```
[ERROR: cannot call json_array_elements on a non-array](https://stackoverflow.com/questions/68702180/error-cannot-call-json-array-elements-on-a-non-array)
```

File: tests/fixtures/models.py [hunk 2]
```diff
@@ -28,7 +28,7 @@ def make_models():
         published=datetime.now(),
         description="",
         ubuntu_description="",
-        notes={},
+        notes=[{'author': 'mysql', 'note': 'mysql-1.2 is not affected by this CVE'}],
```
Comment (carkod) [inline tests/fixtures/models.py:31]: For some reason, tests where picking up this and not the test `PUT`

File: webapp/models.py [hunk 3]
```diff
@@ -57,7 +57,7 @@ class CVE(db.Model):
     updated_at = Column(
         DateTime(timezone=True), server_default=func.now(), onupdate=func.now()
     )
-    notes = Column(JSON)
+    notes = Column(JSON)  # JSON array
```
Comment (carkod) [inline webapp/models.py:60]: Important comment. Do not initialize test data as a JSON object or anything else.

**Review by mtruj013 — APPROVED** [review]
Thanks @carkod , LGTM!

**Pull request merged**